indico/modules/events/editing/client/js/editing/timeline/StateIndicator.module.scss

```diff
@@ -0,0 +1,12 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+.label-text {
+    margin-left: 5px;
```

Comment:
`position: relative;` and `bottom: 2px` are not needed here, as they don't help much when aligning the contents of `StateIndicator` but you need to also add here the three rules you have in the other class (`text-overflow: ellipsis;  white-space: nowrap; overflow: hidden;`). That way the ellipsis dots also appear in the same color as the text.

I'm not sure whether this is the correct way have proper text-wrapping, by having these three rules both in this and the parents class. Maybe someone else knows better.

Reply:
I will do this right away. Thanks for checking this out.